cellpack/autopack/loaders/recipe_loader.py

```diff
@@ -123,6 +124,47 @@ def _migrate_version(recipe):
             recipe["bounding_box"] = recipe["options"]["boundingBox"]
         return recipe
 
+    @staticmethod
+    def _delete_unused_attributes(v1_recipe_data):
+        # ingredients = v1_recipe_data["cytoplasme"]["ingredients"]
+        # for ingredient in ingredients:
+        #     ingredient_data = ingredients[ingredient]
+        #     for attribute in list(ingredient_data):
+        #         if attribute in unused_attributes_list:
+        #             del ingredient_data[attribute]
+        # return v1_recipe_data
+
+        ingredients = v1_recipe_data["cytoplasme"]["ingredients"]
+        filtered_v1_recipe_data = {}
+        for ingredient in ingredients:
```

Comment:
you're already mapping over every ingredient and every attribute in `_get_v1_ingredients` and `_migrate_ingredient`. You just need to add lines 142-143 to line 153 with an elif statement and delete the rest of this function